api/server/sdk/credentials_test.go

```diff
@@ -456,15 +456,79 @@ func TestSdkCredentialEnumerateAWSSuccess(t *testing.T) {
 
 	req := &api.CredentialEnumerateAWSRequest{CredentialId: "test"}
 
+	enumS3test1 := map[string]interface{}{
```

Comment:
Imran, does gomock has ability to run same test case with different input, meaning I will define set of inputs first and each test cases executes against those?